.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+## Description
+
+Describe your request in details.
+
+**Remember, it's better to add something extra than to miss something important.**
+
+## Checklist
```

Comment:
How about adding a reminder to send PR to 'develop' branch ?

Reply:
@halleysouza
Do you agree to resolve this?

Reply:
SGTM